apps/blockchain/scripts/generate_state_tests.exs

```diff
@@ -0,0 +1,2571 @@
+defmodule GenerateStateTests do
```

Comment:
@masonforest  I think it's better to use `.ex` extension. Maybe we can skip this file from running dialyzer

Reply:
Hmmm, I tried adding it to `excluded_paths` in `mix.exs` but it still failed on CI :/

Does anyone know how to make dialyzer ignore a single file, cc: @cernivec ?

Reply:
@masonforest we can add this file to ignore